### PR TITLE
Add auto-detect of attachments extension  [playwright]

### PR DESCRIPTION
**Pull request by elipskiy**
The Playwright Reporter missed attachments whose extensions could not detect from content type. 

Add auto detect file extensions from the file name. 

#### Checklist
- [x] [Sign Allure CLA][cla]
- [x] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

File: packages/allure-js-commons/src/AllureRuntime.ts [hunk 1]
```diff
@@ -48,9 +48,21 @@ export class AllureRuntime {
   }
 
   writeAttachmentFromPath(
-    fromPath: PathLike,
+    fromPath: string,
```
Comment (baev) [inline packages/allure-js-commons/src/AllureRuntime.ts:51]: breaking change, please revert

**Review by baev — CHANGES_REQUESTED** [review]

File: packages/allure-js-commons/src/AllureRuntime.ts (hunk 1 above)
Comment (elipskiy) [reply packages/allure-js-commons/src/AllureRuntime.ts]: fixed

**Comment by vovsemenv** [comment]
@elipskiy some tests are failed. Could you fix it?

**Review by vovsemenv — APPROVED** [review]

**Comment by vovsemenv** [comment]
@baev we ready to go

**Comment by elipskiy** [comment]
@baev please, merge this pr

**Comment by elipskiy** [comment]
@baev @viclovsky

**Comment by baev** [comment]
The PR isn't merged yet due to the following concerns:

1. Attachment file extension isn't really necessary as it will be detected automatically during report generation if correct content type is supplied. 
2. Besides that, we can use file content to detect correct content type (by using magic bytes) -- it's already implemented in both Allure Report & Allure TestOps
3. It makes sense to reuse existing file extension though. But I'm not sure about extname -- there are plenty of complex file extensions like `tar.gz` , and it may be confusing for some users (or not)
4. Why playwright? The main change is located in commons, so you need to add test to commons module.

**Comment by baev** [comment]
fixed via https://github.com/allure-framework/allure-js/pull/956

**Pull request closed without merging**